Code/Framework/AzToolsFramework/AzToolsFramework/Asset/AssetDebugInfo.cpp

```diff
@@ -238,10 +278,11 @@ namespace AzToolsFramework
         if (AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
         {
             serializeContext->Class<AssetFileDebugInfoList>()
-                ->Version(1)
+                ->Version(2)
                 // We are only reflecting the human readable string here because we do not plan on loading this file type
                 // into memory at this time, and this is the easiest way for our customers to read this info
-                ->Field("humanReadableString", &AssetFileDebugInfoList::m_humanReadableString);
+                ->Field("humanReadableString", &AssetFileDebugInfoList::m_humanReadableString)
+                ->Field("invalidAssetHumanReadableString", &AssetFileDebugInfoList::m_humanReadableString);
```

Comment:
The `invalidAssetHumanReadableString` needs to be different variable.
```suggestion
                ->Field("humanReadableString", &AssetFileDebugInfoList::m_humanReadableString)
                ->Field("invalidAssetHumanReadableString", &AssetFileDebugInfoList::m_invalidAssetHumanReadableString);
```